MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,206 @@
+* [`Opacity`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Opacity): Specifies the opacity of shape annotation. This value ranges from 0 to 1.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.AddShape(AnnotationShape.Rectangle, 
+    new ImageEditorShapeSettings() 
+    { 
+        Color = Colors.Blue, 
+        StrokeThickness = 5, 
+        IsFilled =false 
+    });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Delete the selected shape
+
+Delete the selected shape using either the toolbar or the [`DeleteAnnotation`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_DeleteAnnotation) method.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.DeleteAnnotation();
+
+{% endhighlight %}
+{% endtabs %}
+
+## Clear all annotations
+
+Remove all the annotations using the [`ClearAnnotations`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ClearAnnotations) method.
+
+N> It will remove text and pen annotations as well.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.ClearAnnotations();
+
+{% endhighlight %}
+{% endtabs %}
+
+## AnnotationSelected event
+
+The [`AnnotationSelected`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AnnotationSelected) event occurs when an annotation is selected.
+
+N> The event is common for text and shape annotations.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+    <imageEditor:SfImageEditor Source="image.png" AnnotationSelected = "OnAnnotationSelected" />
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="C#" %}
+
+    private void OnAnnotationSelected(object sender, AnnotationSelectedEventArgs e)
+    {
+        if (e.AnnotationSettings is ImageEditorShapeSettings shapeSettings)
+        {
+            shapeSettings.Color = Colors.Black;
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add shape on initial loading
+
+Annotate a shape on image loading using the [`Imageloaded`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageLoaded) event.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+    <imageEditor:SfImageEditor x:Name="imageEditor" Source="image.png" ImageLoaded = "OnImageLoaded" />
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="C#" %}
+
+    private void OnImageLoaded(object sender, EventArgs e)
+    {
+        this.imageEditor.AddShape(AnnotationShape.Circle);
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add shape with manual bounds
```

Comment:
add as sub title
### Add shape with manual bounds

Reply:
Used 3# to sub title